### PR TITLE
Remove the Rails App to Prep for terra-site

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
This PR
- Remove the rails app
- Sets up the repo environment

The next PR will be to consumer terra-site and build the terra-ui site.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ Cerner Corporation
 
 - Matt Butler [@matt-butler]
 - Tao Zhang [@windse7en]
```
Comment (mjhenkes) [inline CONTRIBUTORS.md]: If we're completely re-writing this, are they technically still contributors?

Comment (emilyrohrbough) [reply CONTRIBUTORS.md]: 🤷‍♀️

Comment (emilyrohrbough) [reply CONTRIBUTORS.md]: On this page they would be https://github.com/cerner/terra-ui.com/graphs/contributors

Comment (mjhenkes) [reply CONTRIBUTORS.md]: Not a big deal, more of a philosophical question than anything.

File: Procfile [hunk 2]
```diff
@@ -1 +1 @@
-web: bundle exec puma -C config/puma.rb
+web: npm run start:express
```
Comment (mjhenkes) [inline Procfile]: this file can be deleted

File: CONTRIBUTORS.md (hunk 1 above)
Comment (bjankord) [reply CONTRIBUTORS.md]: Let's leave them as contributors

File: Procfile (hunk 2 above)
Comment (emilyrohrbough) [reply Procfile]: No heroku?

**Pull request merged**